src/test/helpers.ts

```diff
@@ -139,15 +139,29 @@ export function getFirstCallback(mock: jest.Mock, callbackArgIndex: number = 0):
 
 /**
  * Returns the contents of the specified file, if the file is not found returns null
- * @param uri The uri relative to the 'gen' folder specified in EDGE_CHROMIUM_PATH environment variable.
+ * @param uri The uri relative to the 'gen' folder.
  */
 export function getTextFromFile(uri: string) {
+    // Changing from C:\[vscode_directory]\src\test to C:/[vscode_directory]
+    const dirName = removeLastTwoDirectories(__dirname.replace(/\\/g, '/'));
+    const sourceFilesPath = dirName + '/out/edge/src';
+
     const toolsGenDir =
-        `${process.env.EDGE_CHROMIUM_PATH}/out/${process.env.EDGE_CHROMIUM_OUT_DIR}/gen/devtools/`;
+        `${sourceFilesPath}/out/Release/gen/devtools/`;
     const filePath = `${toolsGenDir}${uri}`;
     if (fs.existsSync(filePath)) {
         return fs.readFileSync(filePath, "utf8");
     }
 
     return null;
 }
+
+/**
+ * @param filepath
+ */
+function removeLastTwoDirectories(filepath: string) {
+    const arr = filepath.split('/');

```

Comment:
instead of doing arr.pop() twice, we could use path and select the specific part we are interested in.

Reply:
Looking at the path library, I could not find a function that easily parsed out and selected which directory I wanted.  I improved the code to:
```
    const arr = filepath.split(path.sep);
    arr.splice(-2);
    return arr.join(path.sep);
```

I think this is as simple as I can get it.

Reply:
this is good! thanks Michael. I would post the function I was talking about but I spent 15 minutes looking for it and could not find it... I'll take note if I see it again 👍